File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer =''
    count = 0
    for i in range(len(s)) : 
        print(count)
        if s[i] ==' ' :
            answer += s[i]
            count = 0
            
        elif s[i]!=' ' and count%2 == 0:
            answer += s[i].upper()
            count+=1
            
        else :
            answer += s[i].lower()
            count+=1
    return answer

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int pointer = 0;
        
        for (int i=0; i<s.length(); i++) {
            if(s.charAt(i) != ' '){
                if( (pointer+1) %2 == 1){
                    char upper = Character.toUpperCase(s.charAt(i));
                    answer += upper;
                }else{
                    char downer = Character.toLowerCase(s.charAt(i));
                    answer += downer;
                }
                pointer += 1;
            }else {
                answer += " ";
                pointer = 0;
                
            }
        }
        
        return answer;
    }
}